platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -0,0 +1,648 @@
+/* Copyright (c) 2021, ARM Limited
+ * All rights reserved.
```

Comment:
I wonder if it makes sense to keep duplicating the code that is similar between this new odp_crypto_armv8.c, odp_crypto_null.c and odp_crypto_openssl.c and even with the odp_crypto.c of ODP-DPDK.

Maybe there should be single common top layer of the crypto implementation and then device/platform/implementation specific layer which is were this ARM library adaptation would go. In addition to avoiding code duplication and the resulting added maintenance burden, it would open the door for having more than single crypto implementation enabled at the same time. Note that this revision of this file does not have any ARM specific code.

But maybe such cleanup does not need to be a prerequisite for this patch set. If @MatiasElo is ok with the duplication for now, then I am ok with it too. The cleanup could be done at a later time too when desired.